demo/opae-nlb-demo/Dockerfile

```diff
@@ -28,30 +17,15 @@ RUN cd /usr/src/opae/opae-sdk-${OPAE_RELEASE} && \
     mkdir build && \
     cd build && \
     CFLAGS="$CFLAGS -Wno-misleading-indentation" \
-    cmake -DCMAKE_BUILD_TYPE=Release -DOPAE_BUILD_LIBOPAE_PY=OFF .. && \
-    make xfpga nlb0 nlb3
-
-# Install clean os-core and libstdcpp bundle in target directory
-RUN mkdir /install_root \
-    && swupd os-install  \
-    ${CLEAR_LINUX_VERSION} \
-    --path /install_root --statedir /swupd-state \
-    --bundles=libstdcpp \
-    --no-boot-update \
-    && rm -rf /install_root/var/lib/swupd/*
-
-# Minimal result image
-FROM scratch as final
-ENV PATH="/usr/local/bin/:${PATH}"
-COPY --from=builder /install_root /
-
-# OPAE
-# nlb0 and nlb3 examples
+    cmake .. && \
+    make -j xfpga nlb0 nlb3
+FROM debian:unstable-20210408-slim
```

Comment:
I'd prefer `debian:unstable-slim`.

Reply:
debian:unstable-slim is a moving target. I used specific target to ensure that changes in unstable-slim will not break image build

Reply:
we used to have a moving target too. I'd prefer to keep the base updated (including all the security fixes etc.).

Reply:
done